### ISSUE TITLE
feat(a11y): Add keyboard accessibility to AT node authoring

### ISSUE THREAD

**Issue by breity** [WISE-Community/WISE-Client#1948]
## Description
Some action elements in the Authoring Tool step (node) authoring components are not keyboard accessible. All elements should be navigable and selectable using the keyboard.
